### DISCUSSION TITLE
Fetching Organization members who are owners using GraphQL

### DISCUSSION THREAD

**Discussion by alok-gupta-sada** [community/community#24822] [Apps, API and Webhooks]
<p>How can I find out if the user is an owner of the organization? I am using the following to get a list of members of a given organization -</p>
<p>query ($org: String!, $after: String) {</p>
<p>  organization(login: $org) {</p>
<p>    membersWithRole(first: 100, after: $after) {</p>
<p>      nodes {</p>
<p>        login</p>
<p>        name</p>
<p>      }</p>
<p>      pageInfo {</p>
<p>        hasNextPage</p>
<p>        endCursor</p>
<p>      }</p>
<p>    }</p>
<p>  }</p>
<p>}</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>This is one of the cases where the <code>edges</code> and <code>nodes</code> properties of the connection are more different than simply having a <code>totalCount</code> field. This query should work for you:</p>
<pre><code>query ($org: String!, $after: String) {
  organization(login: $org) {
    membersWithRole(first: 100, after: $after) {
      edges {
        cursor
        node {
          login
          name
        }
        role
      }
    }
  }
}
</code></pre>
<p>I hope that helps!</p>

**Comment by gcs-github-integration** [comment]
<p>Thanks, this is exactly what I was looking for.</p>

**Comment by alok-gupta-sada** [comment]
<p><a class="mention" href="/lee-dohm">@lee-dohm</a> - Is it also possible to get users based on their roles? For example, get me the list of owners for an organization.</p>
<p>Thanks, </p>

**Comment by lee-dohm** [comment]
<p><a class="mention" href="/galok73">@galok73</a> Since there’s no filter options on the <a href="https://developer.github.com/v4/object/organization/#memberswithrole" rel="nofollow noopener"><code>membersWithRole</code> connection on <code>Organization</code></a>, no. You’ll have to get all the users and then filter them yourself.</p>

**Comment by alok-gupta-sada** [comment]
<p>Thanks for confirming.</p>

**Comment by shivam05011996** [comment]
<p><a class="mention" href="/lee-dohm">@lee-dohm</a> The problem here is that the role will be either of ADMIN or MEMBER, in a situation where there are multiple admins how to we filter if the person is the creator/owner of the organization?</p>

**Comment by lee-dohm** [comment]
<p>A member with the <code>ADMIN</code> role is an owner of the organization:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/03b9306c3987c4c9b99b83b3b07f672354fef57c.png" data-download-href="/uploads/short-url/wWcipzvWuAzDbqZ0ZAL7PgloN6.png?dl=1" title="Screen Shot 2019-11-04 at 8.00.43 AM.png"><img src="https://user-images.githubusercontent.com/1038121/181087004-a571a56c-1582-40d4-8e37-f6ea1958e7e8.png" alt="Screen Shot 2019-11-04 at 8.00.43 AM.png" data-base62-sha1="wWcipzvWuAzDbqZ0ZAL7PgloN6" width="690" height="462" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/03b9306c3987c4c9b99b83b3b07f672354fef57c_2_690x462.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/03b9306c3987c4c9b99b83b3b07f672354fef57c_2_1035x693.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/03b9306c3987c4c9b99b83b3b07f672354fef57c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/03b9306c3987c4c9b99b83b3b07f672354fef57c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-11-04 at 8.00.43 AM.png</span><span class="informations">1114×746 37.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>There isn’t any special status for the creator of the organization, they’re simply granted the <code>ADMIN</code> role (which shows as “Owner” in the UI) to begin with. Anyone who is granted the <code>ADMIN</code> role is on equal footing with everyone else who has the <code>ADMIN</code> role.</p>
<p>I hope that helps!</p>
